integrations/amplication/README.md

```diff
@@ -0,0 +1,18 @@
+# amplication
+
+An integration used to import amplication resources into Port.
+
+## Install & use the integration
+
+- [Integration documentation](https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/) *Replace this link with a link to this integration's documentation*
+- Create a secret in your Port application:
+    - In your Port application, click on the *...* button in the top right corner, and select *Credentials*.
+    - In the *Credentials* window, choose the *Secrets* tab.
+    - To create a new secret, click on the *+ Secret* button.
+    - Name the new secret `amplication_token` and save
+- Profit
+
+## Develop & improve the integration
+
+- [Ocean integration development documentation](https://ocean.getport.io/develop-an-integration/)
+- See out [contributing guide](CONTRIBUTING.md)
```

Comment:
can you leave contact information for support, your company support email, or instructions on how to contact you in case of an issue

Reply:
OK, but I didn't see it in any other integration + in the HiBob PR